src/pages/get-started.astro

```diff
@@ -21,7 +21,7 @@ import Servers from '../components/Servers.vue';
       <Servers />
       <p class="c-container-para">
         Don’t see a server you like?
-        <a href="https://github.com/okTurtles/group-income" class="link server-link" target="_blank">
+        <a href="https://github.com/okTurtles/groupincome.org#-instructions-on-how-to-run-your-own-server" class="link server-link" target="_blank">
```

Comment:
Please update this to point to the `group-income` repo's README, and double-check that the hash is correct as I'd be surprised if it started with a `-`...